build.gradle.kts

```diff
@@ -118,7 +118,7 @@ tasks.javadoc.get().mustRunAfter("build")
 tasks.register("releaseProcessingLib") {
     group = "processing"
     dependsOn("clean","build","javadoc", "writeLibraryProperties")
-    finalizedBy("packageRelease")
+    finalizedBy("packageRelease", "copyZipToPdex")
```

Comment:
One note in general, not for this PR, I would like to see somewhat more standardised task naming

Reply:
I think I agree with you (see https://github.com/mingness/processing-library-template/issues/31). Could you be more specific what kind of standard you mean?

Reply:
I think you worded what I saw better in your issue